content/manager/maintenance-mode.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: bt_wiki
+title: Maintenance Mode
+category: Manager
+draft: false
+weight: 900
+
+---
+
+## Overview
+
+Maintenance-Mode is a state in which the Cloudify Manager rejects all requests, and doesn't perform any action (exceptions are detailed [here](#activated)).
+This mode is useful when it is necessary for the Manager to remain in the exact same state for a certain period of time, such as during the ‘Manager Upgrade' process.
+
```

Comment:
switch sentence order.
manager remain unchanged.
shorten certain period of time